packages/bot-web-ui/webpack.config.js

```diff
@@ -155,6 +155,7 @@ module.exports = function (env) {
                 'mobx-react': 'mobx-react',
                 'react-dom': 'react-dom',
                 '@deriv/deriv-charts': '@deriv/deriv-charts',
+                '@deriv/analytics': true,
```

Comment:
what does setting to `true` mean here? because I saw for other packages/libs we are passing their names as the value